cryptoConsole/cryptowatch.py

```diff
@@ -21,84 +21,105 @@
-        if answer != 'y' and answer != "yes":
+def crypto_file(file_path):
+    if platform in ("linux", "linux2", "darwin") and "~" in file_path:
+        file_path = sub("~", expanduser("~"), file_path)
+    if exists(file_path):
+        answer = input("File already exists at %s, overwrite it? [Y/n] "
+                       % file_path)
+        if answer.lower().strip() not in ('y', 'yes'):
             exit()
     try:
-        with open(filePath, 'w+') as file:
+        with open(file_path, 'w+') as file:
             data = getCryptoTable(False, False)
             file.write(data)
             file.write("\n")
     except IsADirectoryError:
         print("Error: the path provided is a directory")
         exit()
-    printHeader()
+    print_header()
 
 
-def consoleLoop():
+def console_loop():
     try:
         while True:
             print(getCryptoTable(True))
             sleep(30)
     except KeyboardInterrupt:
         clear()
-        printHeader()
+        print_header()
+
 
 def main():
-    parser = argparse.ArgumentParser(prog="Cryptowatch",description='Track prices and account balances for bitcoin, ethereum, and litecoin', epilog="By: Alex Epstein https://github.com/alexanderepstein")
-    parser.add_argument("-m", "--monitor",help="Choose which cryptowatch monitor to use")
-    parser.add_argument("-f", "--file", default="", help="Output the current cryptowatch data to the specified file path")
-    parser.add_argument("-c", "--config", action = "store_true", help="Edit the config file for cryptowatch")
-    parser.add_argument("-v", "--version", action="store_true", help="Display the current version of cryptowatch")
+    parser = argparse.ArgumentParser(
+        prog="Cryptowatch",
+        description='Track prices and account balances for bitcoin, '
+                    'ethereum, and litecoin',
+        epilog="By: Alex Epstein https://github.com/alexanderepstein"
+    )
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
+        "-m",
+        "--monitor",
+        choices=('pie', 'rpi', 'console', 'terminal'),
+        help="Choose which cryptowatch monitor to use"
+    )
+    group.add_argument(
+        "-f",
+        "--file",
+        help="Output the current cryptowatch data to the specified file path"
+    )
+    group.add_argument(
+        "-c",
+        "--config",
+        action="store_true",
+        help="Edit the config file for cryptowatch"
+    )
+    group.add_argument(
+        "-v",
+        "--version",
+        action="store_true",
+        help="Display the current version of cryptowatch"
+    )
     args = parser.parse_args()
-    if (args.version or args.config or args.file or args.monitor) and not (bool(args.version) ^ bool(args.config) ^ bool(args.monitor) ^ bool(args.file != "")):
-        print("Error: all options for cryptowatch are mutually exclusive")
-        exit()
+
     if args.version:
         print("Cryptowatch Version 0.0.7")
```

Comment:
Can you change this 0.0.8 I just updated not to long ago

Reply:
@alexanderepstein ,

Sure, no problem.